pkg/controller/node/nodecontroller.go

```diff
@@ -371,6 +402,42 @@ func (nc *NodeController) Run() {
 			}
 		}, nc.nodeMonitorPeriod, wait.NeverStop)
 
+		go wait.Until(func() {
+			nc.evictorLock.Lock()
```

Comment:
I also thought about having different queue for each namespace, but node controller would need to observe changes to namespaces, handle addition/deletion, do you think it's worth it?